### PR TITLE
allow test-utils to be used without jinja2

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

**Comment by andymckay** [comment]
Not creating the client is a real pain, any reason we can't merge this?

File: test_utils/__init__.py [hunk 1]
```diff
@@ -228,6 +233,7 @@ def _pre_setup(self):
         #self._fixture_setup()
         self._urlconf_setup()
         mail.outbox = []
+        self.client = self.client_class()
```
Comment (peterbe) [inline test_utils/__init__.py:236]: Oh. I didn't see this. Had I seen it I would probably not have bothered merging 
https://github.com/jbalogh/test-utils/pull/24

If you want to I can take care of that when I merge this PR. I just need to test it first.

**Comment by peterbe** [comment]
r+ but we need to move the `self.client = ...` line.

**Comment by peterbe** [comment]
merged https://github.com/jbalogh/test-utils/commit/9fa5b26730e36ee390653f7364a74bb7180e9b4c

**Pull request closed without merging**